cocos/editor-support/cocostudio/WidgetReader/GameNode3DReader/GameNode3DReader.cpp

```diff
@@ -386,7 +386,13 @@ namespace cocostudio
             std::string forwardFileData = options->forwardFileData()->path()->c_str();
             std::string backFileData = options->backFileData()->path()->c_str();
 
-            if (leftFileData.empty() || rightFileData.empty() || upFileData.empty() || downFileData.empty() || forwardFileData.empty() || backFileData.empty())
+            if (leftFileData.empty() || rightFileData.empty() || upFileData.empty() || downFileData.empty() || forwardFileData.empty() || backFileData.empty()
```

Comment:
You don't need to check whether the string is empty, if it's empty, it won't pass isFileExist check